bluesky/run_engine.py

```diff
@@ -1377,18 +1379,21 @@ def _read(self, msg):
         if self._bundling:
```

Comment:
Just to underline the intent here, if a plan tries to 'read' without having first asked to 'create', bundling will be `False`. We still go ahead and read the device and send the result back to the plan. The reading can be used by the plan to do adaptive things; it won't be captured by the RunEngine or emitted in a document.

Below this `if` statement, now having established that we _are_ bundling, we should separately check that the `self._read_type` is either `None` or `'regular'`. (I'm assuming that a given Event can only include readings that we invoked using a consistent read type.)

Reply:
interesting. that's a good feature i wasn't aware about, getting a reading without dispatching documents. thanks for the clarification!